src/common/config/init.js

```diff
@@ -3,15 +3,95 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 import { FEATURE_FLAGS } from '../../features/generic_events/constants'
-import { LOG_LEVELS } from '../../features/logging/constants'
 import { isValidSelector } from '../dom/query-selector'
 import { DEFAULT_EXPIRES_MS, DEFAULT_INACTIVE_MS } from '../session/constants'
 import { warn } from '../util/console'
 import { getNREUMInitializedAgent } from '../window/nreum'
 import { getModeledObject } from './configurable'
 
-const nrMask = '[data-nr-mask]'
+/**
+ * @typedef {Object} Init
+ * @property {Object} [ajax]
```

Comment:
Almost every property in this object is optional, so for almost all of these types, they should be noted as optional

`Object` --> `Object=`

Reply:
They're already noted as optional by the `[]`. 
<img width="377" alt="image" src="https://github.com/user-attachments/assets/a16f7482-10e2-40dd-9f6a-9a39106468f3" />

`=` is actually used to denote a default value. The by-effect makes it optional. It's more for `@param` function defs than type defs.

Reply:
got it, didnt know about the [] in jsdoc, makes sense